Cargo.toml

```diff
@@ -8,5 +8,5 @@ debug = true
 lto = true
 
 [workspace.dependencies]
-arrow = {version = "46", features = ["prettyprint", "ffi"]}
+arrow = {version = "51.0.0", features = ["prettyprint", "ffi"]}
```

Comment:
Can you make it less strict by only keeping the major (and maybe also minor) version like `51` or `51.0` ? Same comment for datafusion.